src/test/java/AppointmentTest.java

```diff
@@ -287,16 +287,16 @@ void test_HE_003_TC_12_Valid_Login_From_Hamburger_Menu_Medial_Care_MediaCare() {
         assertEquals(getTextHealthMadicareCheckBox,"Medicare");
     }
     @Test
-    void test_HE_003_TC_13_Valid_Login_From_Hamburger_Menu_Medial_Care_MediaCare() {
-        var getTextHealthMadicareCheckBox = homePage
+    void test_HE_003_TC_13_Valid_Login_From_Hamburger_Menu_Medial_Care_Medicaid() {
+        var getTextHaelthMedicaidCheckBox = homePage
                 .hamburgerMenu()
                 .loginFromHamburgerMenu()
```

Comment:
The name of variable is incorrect "getTextHaelthMedicaidCheckBox"
Should be "TextHaelthMedicaidCheckBox"

Reply:
fixed